titlecase/tests.py

```diff
@@ -330,12 +336,12 @@ def test_input_output():
 
 def test_callback():
     def abbreviation(word, **kwargs):
-        if word.upper() in ('TCP', 'UDP'):
```

Comment:
What's the reason for changing this from `TCP` to `SAML`? (Particularly in lieu of just adding `SAML`?)

Reply:
Oh right, forgot to explain that.  Since this PR make terms that are made purely of consonants be identified as acronyms, it causes [this test](https://github.com/ppannuto/python-titlecase/pull/53/files#diff-0d0f0aea2f094da49701574bc2bbdfdfL336) to fail.  That's because `titlecase` will output `"TCP"` by default now rather than `"Tcp"`.  

Changed to `SAML` since that has a vowel in it, so is immune to this PR's change in logic.

Reply:
Ah, that makes sense. I think I'd actually rather keep `tcp` in there, but just update the test to adapt to this PR's behavior. I view the tests slightly as a self-documenting understanding of what the library will do.

Could you update and add a comment to test that explains something like "the library is able to guess that all-consonant words are acronyms, so TCP works naturally, but others will require the custom list" ?

Reply:
Ah yes, great point, that is better.  I've just made that change.